src/configs/editor-views.ts

```diff
@@ -86,5 +87,6 @@ export default (props: EditorProps, dispatch: Dispatch) => {
     table_element: tableElement(props),
     toc_element: tocElement(props),
     meta_section: metaSection,
+    titles: titles(props),
```

Comment:
no need for that unless there is something special we need to do for title view. and with the class you add to the `toDom` in `manuscripts-transform` you could style the title view